tests/manifests/golden-dv.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Same for the file name - golden-datavolume.yml

Reply:
Done